tree/ntuple/v7/inc/ROOT/RNTupleProcessor.hxx

```diff
@@ -122,35 +128,22 @@ protected:
 
    NTupleSize_t fNEntriesProcessed;  //< Total number of entries processed so far
    std::size_t fCurrentNTupleNumber; //< Index of the currently open RNTuple
-   NTupleSize_t fLocalEntryNumber;   //< Entry number within the current ntuple
+   NTupleSize_t fCurrentEntryNumber; //< Entry number within the current ntuple
```

Comment:
(just for my understanding: this is already `Current processor entry number` as of this commit, right?)